db/dcrpg/upgrades.go

```diff
@@ -78,20 +78,11 @@ var votingMilestones = map[string]dbtypes.MileStone{}
 // toVersion defines a table version to which the pg tables will commented to.
 var toVersion TableVersion
 
-// CheckForAuxDBUpgrade checks if an upgrade is required and currently supported.
-// A boolean value is returned to indicate if the db upgrade was
-// successfully completed.
-func (pgb *ChainDB) CheckForAuxDBUpgrade(dcrdClient *rpcclient.Client) (bool, error) {
-	var version, needVersion TableVersion
-	upgradeInfo := TableUpgradesRequired(TableVersions(pgb.db))
-
-	if len(upgradeInfo) > 0 {
-		version = upgradeInfo[0].CurrentVer
-		needVersion = upgradeInfo[0].RequiredVer
-	} else {
-		return false, nil
-	}
-
+// AddAuxPendingDBUpgrades upgrades all the tables with the pending updates from
```

Comment:
The name of the function is confusing to me.  Since it's a method of `ChainDB`, no need for the "Aux" part.  Also, I'm confused about the "Add" part.  What does that imply?  From the description, I think `UpgradeTables` would be appropriate.